packages/repo-app-import-sub-generator/src/prompts/prompts.ts

```diff
@@ -130,9 +130,8 @@ export async function getPrompts(
                 },
                 message: t('prompts.appSelection.message'),
                 choices: (): { name: string; value: AppInfo }[] => (appList.length ? formatAppChoices(appList) : []),
-                validate: async (answers: AppInfo): Promise<boolean> => {
-                    const result = await validateAppSelection(answers, appList, quickDeployedAppConfig, appWizard);
-                    return !!result;
+                validate: async (answers: AppInfo): Promise<boolean | ValidationLink | string> => {
```

Comment:
```suggestion
                validate: async (answers: AppInfo): Promise<boolean | IValidationLink | string> => {
```
IValidationLink is the interface (type) , ValidationLink is a class that implements the interface.